client/dist/style.css

```diff
@@ -86,23 +74,131 @@
   padding-top: 25px;
 }
 
-.colored {
-  color: #da3743;
-}
+/*CSS for stars*/
 
 .stars-component {
   display: flex;
   flex-direction: row;
-  padding-bottom: 10px;
+  padding-bottom: 4%;
+}
+
+.colored {
+  color: #da3743;
 }
 
 .uncolored {
   color: #6f737b;
 }
 
-.icon-container {
+/*CSS for header*/
+
+.header-container {
+  font-family: 'Josefin Sans', sans-serif;
+  font-weight: bold;
+  margin-left: 10%;
+  margin-right: 10%;
+  margin-top: 15px;
```

Comment:
Can use margin shorthand here:
`margin: top right bottom left;` and `margin: (top and bottom) (right and left);`
`margin: 15px 10% 0;` === `margin-top: 15px;` `margin-bottom: 0px;` `margin-left: 10%` AND `margin-right: 10%`